### PR TITLE
Gtk image menu

### PR DESCRIPTION
**Pull request by raveit65**
With replacing gtkstock i need a bit help.
I am not able to set the icon.
atril_print_menu

### CODE REVIEW

File: shell/ev-sidebar-links.c [hunk 1]
```diff
@@ -334,8 +336,7 @@ build_popup_menu (EvSidebarLinks *sidebar)
 	GtkWidget *item;
 
 	menu = gtk_menu_new ();
-	item = gtk_image_menu_item_new_from_stock ("gtk-print", NULL);
-	gtk_label_set_label (GTK_LABEL (gtk_bin_get_child (GTK_BIN (item))), _("Print…"));
+	item = mate_image_menu_item_new_with_label ("Print…");
```
Comment (zhuyaliang) [inline shell/ev-sidebar-links.c:339]: try to add ```mate_image_menu_item_set_image (MATE_IMAGE_MENU_ITEM (item), gtk-print);```

Comment (raveit65) [reply shell/ev-sidebar-links.c:339]: The first argue is OK , but second failed.
```
ev-sidebar-links.c: In function 'build_popup_menu':
ev-sidebar-links.c:341:70: error: 'gtk' undeclared (first use in this function)
  341 |         mate_image_menu_item_set_image (MATE_IMAGE_MENU_ITEM (item), gtk-print);
      |                                                                      ^~~
ev-sidebar-links.c:341:70: note: each undeclared identifier is reported only once for each function it appears in
ev-sidebar-links.c:341:74: error: 'print' undeclared (first use in this function); did you mean 'printf'?
  341 |         mate_image_menu_item_set_image (MATE_IMAGE_MENU_ITEM (item), gtk-print);
      |                                                                          ^~~~~
```
I add  `GtkWidget *gtk-print;` but it doesn't help.

Comment (zhuyaliang) [reply shell/ev-sidebar-links.c:339]:
```
        GtkWidget *menu;
+       GtkWidget *image;
        GtkWidget *item;
 
        menu = gtk_menu_new ();
        item = mate_image_menu_item_new_with_label ("Print…");
+       image = gtk_image_new_from_icon_name ("gtk-print", GTK_ICON_SIZE_MENU);
+       mate_image_menu_item_set_image (MATE_IMAGE_MENU_ITEM (item), image);
        gtk_widget_show (item);
        gtk_menu_shell_append (GTK_MENU_SHELL (menu), item);
        g_signal_connect (item, "activat
```

Comment (raveit65) [reply shell/ev-sidebar-links.c:339]: yeap, that works, thank you.

**Review by lukefromdc — APPROVED** [review]
Builds and runs fine, actual code change is simple: replacing deprecated gtk code (which could bitrot down the line) with our own. Similar PR's have worked well before

**Pull request merged**